### DISCUSSION TITLE
cannot detect HWW using remote desktop USB connection

### DISCUSSION THREAD

**Discussion by kurosh87** [cryptoadvance/specter-desktop#1496] [Nodeboxes]
Hi
![photo_2021-11-27_13-20-21](https://user-images.githubusercontent.com/95127236/143678472-6fae7005-3d98-42fc-933c-abd5c5636662.jpg)
,

I am trying to connect my ledger wallet to Specter wallet on my Umbrel node. It refuses to detect it. I have run through the remote usb settings and no dice. Anyone that could possibly help? thank you

**Comment by TKX9** [comment]
Dear kurosh87,
I am having exactly the same issue but on a Raspiblitz. Did you find a solution?
